core/domain/classifier_services.py

```diff
@@ -127,6 +127,52 @@ def classify_string_classifier_rule(state, normalized_answer):
     return None
 
 
+def create_classifier_training_jobs(exploration, state_names):
+    """Creates ClassifierTrainingJobModel instances for all the state names
+    passed into the function.
+
+    Args:
+        exploration: Exploration. The Exploration Domain object.
+        state_names: list. List of state names.
+    """
+    exp_id = exploration.id
+    exp_version = exploration.version
+    states = exploration.states
+    for state_name in state_names:
+        state = states[state_name]
+        training_data = state.get_training_data()
+        interaction_id = state.interaction.id
+        algorithm_id = feconf.INTERACTION_CLASSIFIER_MAPPING[
+            interaction_id]['algorithm_id']
+        create_classifier_training_job(algorithm_id, interaction_id,
+                                       exp_id, exp_version, state_name,
+                                       training_data,
+                                       feconf.TRAINING_JOB_STATUS_NEW)
+
+
+def update_classifier_exploration_mappings(exploration, state_names):
+    """Creates new ClassifierExplorationMappingModel instances for all the
+    state names passed into the function.
+
+    Args:
+        exploration: Exploration. The Exploration Domain object.
+        state_names: list. List of state names.
+    """
+    exp_id = exploration.id
```

Comment:
@seanlip There is a small issue here, which I wanted to talk to you about earlier today. I was gonna restructure this part. So, right now, if I don't have a classifier already, I just say 'continue'. Whereas, I need to update it such that the previous job needs to be invalidated and a new job has to be created. I shall try to do this in a separate function and make the update_classifier_mapping method do nothing other than create mappings.